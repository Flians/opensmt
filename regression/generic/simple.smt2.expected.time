Command terminated by signal 11
real 0.13
user 0.00
sys 0.00
