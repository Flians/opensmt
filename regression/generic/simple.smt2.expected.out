(error "unknown logic AB_CD")

